component-test/test_dbus_proxy.py

```diff
@@ -521,7 +521,7 @@ class TestProxyMethods(object):
             "direction": "*",
             "interface": "*",
             "object-path": "*",
-            "method": ["Method1", "Method3", "Method4", "Method2"]
+            "method": ["Method1", "Method3", "Method2", "Method4"]
```

Comment:
To prove that the order of methods in a list is irrelevant. New test is added to examine this to fulfill concerns.